packages/react-native-reanimated/src/common/types.ts

```diff
@@ -0,0 +1,6 @@
+'use strict';
+export type Maybe<T> = T | null | undefined;
+
+export type ValueProcessor<V, R = V> = (
+  value: V
+) => Maybe<R> | Record<string, R>;
```

Comment:
```suggestion
) => Maybe<R>;
```

This `Record<string, R>` type is used only in CSS animations where we split property into a few separate props, e.g. `marginHorizontal` to `marginLeft` and `marginRight`.

Reply:
Yet, as this is a common type, we won't need it to handle css?